File: frontend/henn-laszlo/projects/artwork-data-access/src/lib/models/admin-artwork-list-item.ts
export interface AdminArtworkListItem {
  readonly id: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly year: number | null;
  readonly techniqueHu: string | null;
  readonly widthCm: number | null;
  readonly heightCm: number | null;
  readonly hasImage: boolean;
  readonly isPublished: boolean;
  readonly isFeatured: boolean;
  readonly displayOrder: number;
  readonly createdAtUtc: string;
}

File: backend/src/Modules/Modules.Artwork.Application/Artworks/GetAdminList/AdminArtworkListItem.cs
namespace Modules.Artwork.Application.Artworks.GetAdminList;

public sealed record AdminArtworkListItem(
    Guid Id,
    string TitleHu,
    string? TitleEn,
    int? Year,
    string? TechniqueHu,
    decimal? WidthCm,
    decimal? HeightCm,
    bool HasImage,
    bool IsPublished,
    bool IsFeatured,
    int DisplayOrder,
    DateTime CreatedAtUtc);